docs/linux_dependencies.md

```diff
@@ -136,15 +136,15 @@ You can do this in one line with `nix-shell --run "cargo run"`.
 
 This is also possible with [Nix flakes](https://nixos.org/manual/nix/unstable/command-ref/new-cli/nix3-flake.html).
 Instead of creating `shell.nix`, you just need to add the derivation (`mkShell`)
-to your `devShells` in `flake.nix`. Run `nix develop` to enter the shell and
+to your `devShells` in `flake.nix`. Run `nix develop` to enter the shell or just
 `nix develop -c cargo run` to run the program. See
 [Nix's documentation](https://nixos.org/manual/nix/unstable/command-ref/new-cli/nix3-develop.html)
 for more information about `devShells`.
 
 Note that this template does not add Rust to the environment because there are many ways to do it.
 For example, to use stable Rust from nixpkgs, you can add `cargo` and `rustc` to `nativeBuildInputs`.
 
-[Here](https://github.com/NixOS/nixpkgs/blob/master/pkgs/games/jumpy/default.nix)
+[Here](https://github.com/NixOS/nixpkgs/blob/e0464e47880a69896f0fb1810f00e0de469f770a/pkgs/games/jumpy/default.nix)
```

Comment:
This file was moved to https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/ju/jumpy/package.nix